R/ff_prep.R

```diff
@@ -165,25 +165,34 @@ ff_prep <- function(datafolder = NA, country = NA, shape = NA, tiles = NULL, gro
       rm(extent)
     }
     files <- allfiles[grep(tile, allfiles)]
+    # initialize_shape_from_borders
     if (!hasvalue(shape) & (shrink == "extract")) {
       if (!exists("countries", inherits = F)) {
         data(countries, envir = environment())
         borders <- terra::vect(countries)
       }
       shape <- aggregate(intersect(terra::as.polygons(terra::ext(terra::rast(files[1]))), borders))
     }
+    # end-of initialize_shape_from_borders
+
+    # process_tile_dates (process_a_tile_in_dates)
```

Comment:
why this comment?

Reply:
I am trying to fully run ff_prep_refactored. Fyi. it still has an error until now. 
To fix the errors, I need to compare the refactored functions with the original code. With those comments, I can easily see the original code for each refactored functions.

Reply:
but you dont need to merge them to develop, have them locally

Reply:
you're kinda ignoring the fact that we have to fix ff_prep_refactored.
not only me that need this